### PR TITLE
Launch Galactic Recursion

### PR DESCRIPTION
**Pull request by darabos**
For https://github.com/biggraph/biggraph/issues/3024.

### CODE REVIEW

**Comment by xandrew-lynx** [comment]
LGTM, but let's wait for Gergo's opinion before actually merging.

**Comment by darabos** [comment]
Please take another look. I've refreshed Santa's Little Helper a bit. I've carried over the improvements made in Galactic Recursion, such as ignoring results from earlier service workers, the reactive font sizing, and the nice title font. I've also added the mission choice and referenced the two missions from each other. It's live on `ramhegy:8000`.

**Comment by darabos** [comment]
> Please take another look.

GitHub is kind of having an outage, and the changes are pushed, but not visible on the web UI. I hope it's all fixed by tomorrow!

**Comment by darabos** [comment]
> GitHub is kind of having an outage, and the changes are pushed, but not visible on the web UI. I hope it's all fixed by tomorrow!

I had to add another commit (just a merge) and now everything showed up. Good for another look then.

File: hiring/santa.html [hunk 1]
```diff
@@ -29,14 +34,14 @@
...
       <a href="/hiring/">Lynx Analytics R&amp;D applicants</a>.
-      <!--
-      Our current hiring exercise is <a href="/hiring/galactic.html">Galactic Recursion</a>.
-      -->
+      Do not let that stop you, though, even if you have no intention of applying!
       <p>You can solve the exercises in JavaScript in the browser.
       Or you can solve them with whatever technology you prefer and then simply return the solution from the JavaScript function.
-      In this case it would be nice of you to include your code in a comment above the return statement.
+      In this case it would be nice of you to describe your solution in a comment above the return statement.
+      <p>If you don't like the cold, check out our other hiring exercise,
```
Comment (xandrew-lynx) [inline hiring/santa.html]: I read the spaceship is cold to. Maybe snow?

**Comment by xandrew-lynx** [comment]
LGTM

File: hiring/santa.html (hunk 1 above)
Comment (darabos) [inline hiring/santa.html]: > I read the spaceship is cold to. Maybe snow?
 
Done, thanks!

**Comment by darabos** [comment]
@gsvigruha can only do it on the weekend. Good to go then?

You mentioned sending this to US interns? Should I switch the job description from Hungarian to English?

**Comment by gsvigruha** [comment]
> Should I switch the job description from Hungarian to English?

Maybe that would be good in general, in case we want to use this at some point in Singapore too.

**Comment by darabos** [comment]
> > Should I switch the job description from Hungarian to English?
>
> Maybe that would be good in general, in case we want to use this at some point in Singapore too.

Okay, I've switched to English. I'm happy to revert it if you'd like to keep it Hungarian, or please review the translation if you want the English version. Or if you want both. We could have some sort of switch.

File: hiring/index.html [hunk 2]
```diff
@@ -27,59 +27,54 @@
...
+    <p><b>You can expect:</b>
     <ul>
-      <li>egy ambíciózus termékfejlesztésbe kapcsolódna be, de nem csak programozásból állna a munka,
-      hanem gondolkodós, matematikai &mdash; főleg algoritmus- és gráfelméleti &mdash; jellegű problémákat is meg kellene oldani
+      <li>To join us on an ambitious product development project. Besides programming, the work
+      includes thinking and mathematics &mdash; mainly algorithmic and graph-theoretic problems.
 
-      <li>a szoftver tervezésében, implementációjában négy veterán fejlesztővel (ex-Googler, ex-NNG) dolgozna együtt,
-      míg az elméleti kihívásokat amerikai vezető egyetemeken
-      (Columbia, Berkeley, University of Florida, National University of Singapore) kutató professzorokkal együttműködve kellene megoldania
```
Comment (xandrew-lynx) [inline hiring/index.html]: Who is working on NUS? I understand this was not introduced now but I have no idea who we had in mind back then. Unless somebody remembers, let's remove it.

**Comment by xandrew-lynx** [comment]
Still LGTM. We won't be sending the job description to the interns, only the test, so the language doesn't matter for that. (This job description does not apply to them anyways.) But English is fine, too.

Let's launch tomorrow. I am still trying to get Judit to have a go, at least up to understanding the problems.

@Gergo, can you also maybe have such a quick go (if you haven't yet)? Use #reveal (you may need an extra refresh after adding it) to see the full stuff without solving anything. The aim is to test that the problems are clear.

File: hiring/index.html (hunk 2 above)
Comment (darabos) [inline hiring/index.html]: > Who is working on NUS? I understand this was not introduced now but I have no idea who we had in mind back then. Unless somebody remembers, let's remove it.

It was added by Gyuri in 2014 in the Google Docs draft. (https://docs.google.com/document/d/1ghPtkpi7Yv4xTD-6xPx7dMXxqzFO8PlvCDnR1iw0puU/edit?disco=AAAAAMJGZOk) May not be accurate anymore. I've removed it.

**Comment by darabos** [comment]
I've removed the "Hamiltonian" problem per discussion. Can you please check if the story still makes sense? Thanks!

**Comment by xandrew-lynx** [comment]
LGTM, but needs a manual merge.

**Pull request merged**